### ISSUE TITLE
Enigmatic error messages when uploading the package to CRAN

### ISSUE THREAD

**Issue by weirichs** [weirichs/eatTools#40]
When I upload the package to CRAN with `devtools::release()`, I receive the following error message in the last step, which I cannot assign. However, the package is still uploaded to CRAN and everything runs as intended. 
![Unbenannt](https://github.com/user-attachments/assets/fa2b26f3-1200-423f-bff2-9f31eb49e060)

**Comment by beckerbenj** [comment]
Are you aware that you removed the `cran-comments.md` in your last commit? This may have caused these issues with `devtools`. Also, I think CRAN has a strong preference for incrementing the version number if one submits a new package version (which I would argue is the case here).

**Comment by weirichs** [comment]
But the version number has been upgraded from 0.7.7 to 0.7.8. There was no version 0.7.8 on CRAN until now. And yes, I should have recreated the CRAN comments beforehand with `usethis::use_cran_comments()`, I had forgotten.

**Comment by beckerbenj** [comment]
> But the version number has been upgraded from 0.7.7 to 0.7.8. There was no version 0.7.8 on CRAN until now.

That is true, but submitting a changed version under `0.7.7` indicates to CRAN that nothing has changed, which is not true. See the CRAN submission policy: https://cran.r-project.org/web/packages/policies.html#Re_002dsubmission

> Increasing the version number at each submission reduces confusion so is preferred even when a previous submission was not accepted. 

I don't think this as a rejection reason for CRAN, but maybe helpful for future submissions.

**Comment by beckerbenj** [comment]
Regarding the issue: 
I would hope that this will be fixed automatically when you recreate the `cran-comments.md` (but is hard to test before the next CRAN release I guess?). I hope it is ok if I assign you to the issue and you close it after you have added it?

And thanks for taking care of the CRAN release of `eatTools`!

**Comment by beckerbenj** [comment]
Can this be closed @weirichs?
